toolkit/tools/pkg/imagecustomizerlib/customizeverity.go

```diff
@@ -63,7 +63,7 @@ func updateFstabForVerity(imageChroot *safechroot.Chroot) error {
 	for _, entry := range fstabEntries {
 		if entry.Target == "/" {
 			// Replace existing root partition line with the Verity target.
-			entry.Source = "/dev/mapper/root"
+			entry.Source = imagecustomizerapi.VerityRootDevicePath
```

Comment:
Perhaps we can also pass `rootfsVerity` as parameter of function `updateFstabForVerity`, then give `entry.Source = "dev/mapper/" + rootfsVerity.Name`. This way it can support `usr` Verity as well though for now we only support `rootfs` Verity, thoughts?

Reply:
Fixed.